src/globus_sdk/experimental/tokenstorage/base.py

```diff
@@ -148,3 +180,47 @@ def user_only_umask(self) -> t.Iterator[None]:
             yield
         finally:
             os.umask(old_umask)
+
+
+def _default_globus_app_filename(
```

Comment:
This isn't really the _filename_, though—it's the full path, right?

Reply:
Sure, I can rename.